pkg/resources/model/repository.go

```diff
@@ -334,6 +337,25 @@ func (r LoggingResourceRepository) FluentbitsFor(ctx context.Context, logging v1
 	return res, nil
 }
 
+func (r LoggingResourceRepository) LoggingRoutesFor(ctx context.Context, logging v1beta1.Logging) ([]v1beta1.LoggingRoute, error) {
+	var list v1beta1.LoggingRouteList
+	if err := r.Client.List(ctx, &list); err != nil {
+		return nil, err
+	}
+
+	sort.Slice(list.Items, func(i, j int) bool {
+		return lessByNamespacedName(&list.Items[i], &list.Items[j])
+	})
+
+	var res []v1beta1.LoggingRoute
+	for _, i := range list.Items {
+		if i.Spec.Source == logging.Spec.LoggingRef {
+			res = append(res, i)
+		}
+	}
```

Comment:
sorting could be done _after_ filtering so only the "important" items are sorted
```suggestion
	var res []v1beta1.LoggingRoute
	for _, i := range list.Items {
		if i.Spec.Source == logging.Spec.LoggingRef {
			res = append(res, i)
		}
	}

	sort.Slice(res, func(i, j int) bool {
		return lessByNamespacedName(&res[i], &res[j])
	})
```

Reply:
@siliconbrain changed for all the occurences in b8bdce02

Reply:
sorry, its 7ca6841a